components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/dto/OperationPolicyDTO.java

```diff
@@ -0,0 +1,157 @@
+package org.wso2.carbon.apimgt.internal.service.dto;
```

Comment:
Let's add the license header and remove wildcard imports.

Reply:
This is a swagger generated file. Should we modify this class as well?

Reply:
Yes that's right, no need to add the header here as it is an auto generated file.